src/main/java/seedu/student/storage/JsonSerializableStudentBook.java

```diff
@@ -73,7 +73,7 @@ public StudentBook toModelType() throws IllegalValueException {
                 throw new IllegalValueException(MESSAGE_MISSING_STUDENT);
             }
 
-            if (studentBook.hasAppointment(appointment) || studentBook.hasOverlappingAppointment(appointment)) {
+            if (studentBook.hasOverlappingAppointment(appointment)) {
                 throw new IllegalValueException(MESSAGE_DUPLICATE_APPOINTMENT);
             }
```

Comment:
Should the error message talk about clashes or duplication?

Reply:
Good point! Seem more appropriate to talk about appointment clashes. Thanks!